core/keyboard_nav/cursor_svg.js

```diff
@@ -511,6 +511,12 @@ Blockly.CursorSvg.prototype.draw = function(curNode) {
   } else if (curNode.getType() === Blockly.ASTNode.types.STACK) {
     this.showWithStack_(curNode);
   }
+
+  // Ensures the cursor will be visible immediately after the move.
+  var animate = this.currentCursorSvg.childNodes[0];
+  if (animate !== undefined) {
+    animate.beginElement();
```

Comment:
It looks like this line will break things in ie and edge since they don't support animate. So change this to animate.beginElement && animate.beginElement() and then this should be good to go!

Reply:
Thanks @alschmiedt!

Reply:
Thanks for contributing!